features/robots.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Site blocks web spiders
+
+  Scenario: Web spider looks for /robots.txt 
+    Given I am an unauthenticated user
+    When I visit '/robots.txt'
+    Then I see 'User-Agent: *'
```

Comment:
According to robots.txt spec at Google (https://developers.google.com/webmasters/control-crawl-index/docs/robots_txt) the fields are case-insensitive - so all should be well.